docs/variables.md

```diff
@@ -184,6 +184,18 @@ Only available for `successCondition`
 | `response.body` | Response body (`string`) |
 | `response.headers` | Response headers (`map[string][]string`) |
 
+### CronWorkflow Templates
+
+> v3.6 and after, note that the the variables here can be used with the full power of expr-lang expression.
+
+| Variable | Description|
+|----------|------------|
+| `cronworkflows.name` | Name of the CronWorkflow method (`string`) |
```

Comment:
```suggestion
| `cronworkflow.name` | Name of the CronWorkflow (`string`) |
```

This should be singular, it only references a single CronWorkflow. That would also match the `workflow` variables

Also there isn't a "method" here?

Reply:
Thanks, I fixed this